apps/src/templates/studioHomepages/TeacherHomepage.jsx

```diff
@@ -281,14 +284,27 @@ export const UnconnectedTeacherHomepage = ({
             isProfessionalLearningCourse={true}
           />
         )}
+        {/* TODO - We will eventually remove this section
+          once enough time has passed */}
         {(plCourses?.length > 0 || topPlCourse) && (
-          <RecentCourses
-            courses={plCourses}
-            topCourse={topPlCourse}
-            showAllCoursesLink={true}
-            isProfessionalLearningCourse={true}
-            hasFeedback={hasFeedback}
-          />
+          <section id={'pl-courses-placeholder'} style={{marginBlock: '6rem'}}>
+            <Heading2 visualAppearance="heading-md">
```

Comment:
Made this look like an `h3` to match the other headings on this page.

Reply:
Wondering if we should use the `ContentContainer` component like what's being used in `TeacherSections.jsx` to match the other titles on the page?